File: 2956-find-common-elements-between-two-arrays/2956-find-common-elements-between-two-arrays.cpp
class Solution {
public:
    vector<int> findIntersectionValues(vector<int>& nums1, vector<int>& nums2) {
        int count1 = 0 ;
        int count2 = 0;
        int n = nums1.size() ;
        int m = nums2.size() ;
        for (int i=0; i<n; i++){
            for (int j=0; j<m; j++){
                if (nums1[i] == nums2[j]){
                    count1++ ;
                    break ;
                }
            }
        }
        for (int j=0; j<m; j++){
            for (int i=0; i<n; i++){
                if (nums2[j] == nums1[i]){
                    count2++ ;
                    break ;
                }
            }
        }
        vector<int> answer ; 
        answer.push_back(count1) ;
        answer.push_back(count2) ;
        return answer ;
    }
};

File: 2956-find-common-elements-between-two-arrays/2956-find-common-elements-between-two-arrays.py
class Solution:
    def findIntersectionValues(self, nums1: List[int], nums2: List[int]) -> List[int]:
        count1=0
        count2=0
        for i in nums1:
            if i in nums2:
                count1+=1
        for i in nums2:
            if i in nums1:
                count2+=1
        return [count1,count2]
        